### ISSUE TITLE
Font name variables supposed to work?

### ISSUE THREAD

**Issue by arxpoetica** [jonathantneal/postcss-advanced-variables#40]
It broke somewhere after version `1.2.2`...? Not sure if it was in `2.0.0.` or `2.1.0`. The following use to translate:

```scss
$poppins: 'Poppins', 'Helvetica Neue', Helvetica, Arial, sans-serif;
body {
    font-family: $poppins;
}

[to]

body {
    font-family: 'Poppins', 'Helvetica Neue', Helvetica, Arial, sans-serif;
}
```

but now it's outputting like so:

```scss
body {
    font-family: ('Poppins','Helvetica Neue',Helvetica,Arial,sans-serif);
}
```

Any ideas? Might need to roll back until fixed.

**Comment by arxpoetica** [comment]
Update: it was version `2.0.0` that broke it.

**Comment by MrSlide** [comment]
I'm also getting this issue since the upgrade.

**Comment by jonathantneal** [comment]
Thanks for bringing this to my attention, @arxpoetica, all. This is fixed in [v2.2.0](https://github.com/jonathantneal/postcss-advanced-variables/blob/master/CHANGELOG.md#220-january-2-2018).

I’ve also added [tests](https://github.com/jonathantneal/postcss-advanced-variables/blob/master/test/variables.css) for the behavior, which you can compare to the [results](https://github.com/jonathantneal/postcss-advanced-variables/blob/master/test/variables.expect.css).
